src/app/dim-ui/WeaponCatalystInfo.m.scss

```diff
@@ -5,23 +5,27 @@
   padding-top: 2px;
   padding-bottom: 2px;
 
-  color: $orange;
+  color: var(--theme-accent-primary);
```

Comment:
Since this is for Exotic weapon catalysts, perhaps we should use `$exotic` here instead?